gravitee-apim-console-webui/package.json

```diff
@@ -19,10 +19,10 @@
     "@danielmoncada/angular-datetime-picker": "^16.0.1",
     "@danielmoncada/angular-datetime-picker-moment-adapter": "^3.0.1",
     "@fontsource/libre-franklin": "4.4.5",
-    "@gravitee/ui-analytics": "7.30.0",
+    "@gravitee/ui-analytics": "7.33.0",
     "@gravitee/ui-components": "3.44.0",
-    "@gravitee/ui-particles-angular": "7.30.0",
-    "@gravitee/ui-policy-studio-angular": "7.30.0",
+    "@gravitee/ui-particles-angular": "^7.34.0-form-cron-p2-b51c899",
+    "@gravitee/ui-policy-studio-angular": "7.33.0",
```

Comment:
```suggestion
    "@gravitee/ui-particles-angular": "^7.35.0",
    "@gravitee/ui-policy-studio-angular": "7.35.0",
```

Reply:
haha yes of course, i've juste merge ui-p yesterday.